alpha-build-machinery/make/lib/golang.mk

```diff
@@ -12,7 +12,7 @@ GOFMT ?=gofmt
 GOFMT_FLAGS ?=-s -l
 GOLINT ?=golint
 
-GO_FILES ?=$(shell find . -name '*.go' -not -path '*/vendor/*' -not -path '*/_output/*' -print)
```

Comment:
why not `find . -name '*.go' -not -path '*/vendor/*' -not -path '*/_output/*' -not -path '*/.git/*' -print`?

Reply:
Honestly the version provided by Tomas doesn't work for me if I invoked:
```
go list -f '{{range .GoFiles}}{{ printf "%s/%s " $.Dir . }}{{end}}{{range .CgoFiles}}{{ printf "%s/%s " $.Dir . }}{{end}}{{range .TestGoFiles}}{{ printf "%s/%s " $.Dir . }}{{end}}' ./cmd/... 
```
I'm only getting:
```
/home/maszulik/workspace/origin/src/github.com/openshift/oc/cmd/oc/oc.go /home/maszulik/workspace/origin/src/github.com/openshift/oc/cmd/oc/oc_test.go
```
Stefan's version works, although both are ugly to read :sob:

Reply:
`go list` is very powerful, but you need to tweak it a lot to get what you need, let's keep it reasonably simple :wink: